src/components/chart/utils/formatChartData.ts

```diff
@@ -0,0 +1,25 @@
+// @ts-ignore
+import date from "date-and-time";
+
+interface Item {
+  minute: string, 
+  date: string, 
+  close: number
+}
+
+export function formatChartData(data: [] = []) {
```

Comment:
It's better to specify here the type of the array - else `data` is just  `any[]`

If you do it this way, it gets easier to understand what does this function take as parameters, and then you won't need to specify it inside the map below, so you can just do
```
data.map(item => {
```

(typescript can figure it out then)

Reply:
Added to project now